app/views/schools/dashboard/_school_cohort_details.html.erb

```diff
@@ -65,7 +67,9 @@
 <% if latest_partnership&.challenged? == false %>
   <% if latest_partnership&.in_challenge_window? %>
     <p class="govuk-body">
-      If your school is not working with this lead provider or delivery partner, <%= govuk_link_to("tell us before #{latest_partnership.challenge_deadline&.to_date&.to_fs(:govuk)}", challenge_partnership_path(partnership: latest_partnership), id: "challenge-partnership-link") %>. 
+      If your school is not working with this lead provider or delivery
+      partner, <%= govuk_link_to("tell us before #{latest_partnership.challenge_deadline&.to_date&.to_fs(:govuk)}", challenge_partnership_path(partnership: latest_partnership), id: "challenge-partnership-link") %>
+      .
```

Comment:
Did you meant to change this? I think the full stop has to remain on the previous line to avoid a space?

```suggestion
      partner, <%= govuk_link_to("tell us before #{latest_partnership.challenge_deadline&.to_date&.to_fs(:govuk)}", challenge_partnership_path(partnership: latest_partnership), id: "challenge-partnership-link") %>.
```